packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -229,6 +240,41 @@ export class EdgelessPageBlockComponent extends BlockElement<
     return this._viewportElement;
   }
 
+  get viewport(): EdgelessViewport {
+    if (!this.viewportElement) {
```

Comment:
If the `viewportElement` does not exist, I think it should throw an error directly, so as to maintain consistency with the implementation of `get viewportElement()`.

Reply:
@Flrande Done
thanks for the quick review